app/views/spree/admin/reviews/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :page_title do %>
-	<%= I18n.t("spree.editing_review_for_html", link: link_to(@review.product.name, product_path(@review.product))) %>
+	<%= I18n.t("spree.editing_review_for_html", link: @review.product.name) %>
```

Comment:
I think the `link` placeholder is a bit misleading here. We either rename it or make the link work. 🙂

Reply:
It is fixed. I renamed it.